operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/Controller.java

```diff
@@ -71,15 +72,22 @@ public Controller(Reconciler<P> reconciler,
     eventSourceManager = new EventSourceManager<>(this);
 
     final var hasDeleterHolder = new boolean[] {false};
```

Comment:
Why is this not a simple boolean?

Reply:
Because you cannot use a non-final var in a closure.

Reply:
can we use `AtomicBoolean` ?